format.go

```diff
@@ -459,6 +459,7 @@ func (h *DefaultEventHandler) OnReceiveHeaders(md metadata.MD) {
 func (h *DefaultEventHandler) OnReceiveResponse(resp proto.Message) {
 	h.NumResponses++
 	if h.verbose {
+		fmt.Fprintf(h.out, "\nResponse size (bytes): %d\n", proto.Size(resp))
```

Comment:
This is not necessarily the serialized size of the message. This is the size of the _canonical_ serialized size. It is possible that the actual bytes received on the wire differ from this. In fact, the on the wire data could have even been compressed.

And, unfortunately, it's not as simple to capture that real size. It is easier to capture the decompressed size, prior to unmarshaling, via a `ForceCodec` call option. Although that would require more intrusive changes to this package. Getting at the actual on-the-wire size is harder due to how compressors are registered in the gRPC Go runtime. I think the `main` package would have to register a custom compressor that uses the same type/scheme, but then wraps the actual compressor implementation, just to intercept the actual on-the-wire byte slice, to measure its size.